openml/flows/flow.py

```diff
@@ -51,10 +52,16 @@ def _generate_flow_xml(self):
             Flow represented as XML string.
         """
         model = self.model
+
+        if config._testmode:
+            flow_name = '%s%s' % (config.testsentinel, self.name)
```

Comment:
To change the name to be unique for each test run. I found this to be more reliable than deleting runs on the test server. Would it help to add a method `get_name` which I mock during test time?